federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -22,7 +22,11 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
+        {{if eq .KUBERNETES_PROVIDER "aws"}}
+        - --external-hostname={{.FEDERATION_API_HOST}}
```

Comment:
IIUC, the DNS name is cluster internal only? Doesnt seem right to set it as external-hostname.

cc @lavalamp